pkg/runtime/wfengine/wfengine_test.go

```diff
@@ -862,9 +874,15 @@ func startEngineAndGetStore(ctx context.Context, t *testing.T, r *task.TaskRegis
 	return client, engine, store
 }
 
-func getEngine(t *testing.T) *wfengine.WorkflowEngine {
+func getEngine(t *testing.T, ctx context.Context) *wfengine.WorkflowEngine {
```

Comment:
nit: it's more conventional to have `ctx` be the first parameter in Go functions.

Reply:
This is a tricky case because `testing.T` is _also_ supposed to be the first parameter. I believe when we have both, we keep `testing.T` first and then `context.Context`.

Reply:
Ah, good point - I didn't even look carefully to see what the first parameter was. 😅 I agree, we should keep `testing.T` first.